docs/resources/ui-building-blocks/widgets/widget-commonalities.md

```diff
@@ -428,4 +428,78 @@ To add a border to any widget:
         allow="clipboard-write">
     </iframe>
 </div>
+<p></p>
+
+## Styling text
```

Comment:
We already have a section for styling text in text.md, why do we need this here?